src/devices/RadioTransmitter/Devices/Kt0803/TransmissionPower.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.RadioTransmitter
+{
+    /// <summary>
+    /// Transmission Power
+    /// </summary>
+    public enum TransmissionPower : byte
+    {
+        /// <summary>
+        /// 95.5 dBuV
+        /// </summary>
+        Power_095_5dBuV = 0,
```

Comment:
Please remove first underscore (second one is fine since it's separating numbers)